__tests__/marker/with-scope.ts

```diff
@@ -0,0 +1,26 @@
+import { marker } from '@ngneat/transloco-keys-manager/marker';
```

Comment:
the import here should be `@jsverse/transloco-keys-manager/marker` 😉

Reply:
Without my previous [PR](https://github.com/jsverse/transloco-keys-manager/pull/182) this doesn't work :wink:

Reply:
ah you're right, well let's hope that other PR get's resolved 😀

Reply:
Rebased, and done :wink: